docs/koji.md

```diff
@@ -88,6 +88,17 @@ Data which is placed here includes:
 
 - `build.extra.typeinfo.operator-manifests.archive` (string): name of the archive containing operator manifest files
 
+# Type-specific source container build metadata
+
+The same note about `image` and `build.extra.typeinfo.image` from previous section applies here.
+
+Data which is placed here includes:
+
+- `build.extra.image.media_types`: the same as for image builds (see previous section)
+- `build.extra.image.sources_for_nvr` (str): koji N-V-R for the original image build
+- `build.extra.image.sources_signing_intent` (str): signing intent of the ODCS composes
```

Comment:
Is this accurate? I am not sure how to improve the wording here though. Would it be worth to mention this is the signing intent used to fetch the source RPMs?

Since the signing intent here can be different from the one used for the original image build composes, this could sound ambiguous.